src/coreclr/vm/gcenv.ee.common.cpp

```diff
@@ -317,16 +332,21 @@ StackWalkAction GcStackCrawlCallBack(CrawlFrame* pCF, VOID* pData)
     #endif // _DEBUG
 
             DWORD relOffsetOverride = NO_OVERRIDE_OFFSET;
-#if defined(FEATURE_EH_FUNCLETS) && defined(USE_GC_INFO_DECODER)
+#if defined(FEATURE_EH_FUNCLETS)
             if (pCF->ShouldParentToFuncletUseUnwindTargetLocationForGCReporting())
             {
+                bool wantsReportOnlyLeaf = true;
```

Comment:
Nit: This is always set by RyuJIT. We can delete all places that check it in the runtime and change this

https://github.com/dotnet/runtime/blob/8115429a72fa599584135f217f3a34bd4aef8cf4/src/coreclr/vm/gcinfodecoder.cpp#L675

to `#if defined(TARGET_AMD64) && defined(DECODE_OLD_FORMATS)`.

Reply:
Sounds reasonable. It will likely need some documentation/comments update too.